src/pages/landing/stocks/_dtrading.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import device from 'themes/device'
+import { Container, SectionContainer } from 'components/containers'
+import { Header, Text, QueryImage } from 'components/elements'
+
+const StyledSection = styled(SectionContainer)`
+    background-color: var(--color-white);
+    @media ${device.tabletL} {
+        padding: 1.74rem 0 4rem 0;
```

Comment:
You can declare css units for media queries in `px` as 1rem no longer equals 10px for `laptop` or lower widths.

Reply:
already changed to px.